electron/main.ts

```diff
@@ -280,14 +285,15 @@ ipcMain.on('unlock', () => {
 })
 
 ipcMain.on('kill', (event, game) => {
+  Logger.info({message: `Attempting to kill game ${game} `, service: 'kill game'})
```

Comment:
I don't think service name should contain spaces. Seems a bit wrong to do.

Reply:
I saw this when writing it myself and was pretty sure I change it to just kill, must have slipped past me